app/src/modules/settings/routes/flows/components/logs-drawer.vue

```diff
@@ -72,15 +72,14 @@ const steps = computed(() => {
 
 <template>
 	<v-drawer
-		:model-value="!!revisionId"
+		:model-value="!!revision"
 		:title="revision ? revision.timestampFormatted : t('logs')"
 		icon="fact_check"
-		@cancel="emit('close')"
-		@esc="emit('close')"
+		@cancel="closeDrawer()"
+		@esc="closeDrawer()"
```

Comment:
Let’s stick with the `emit` approach

```suggestion
		@cancel="emit('close')
		@esc="emit('close')
```

Reply:
done